report/src/main/webapp/app/services/endpointConfiguration.service.js

```diff
@@ -16,17 +16,16 @@
  * limitations under the License.
  */
 define(['angularAMD'], function (angularAMD) {
-	'use strict';
-	angularAMD.factory('configService', function () {
-		return {
-			getConfig: function () {
-				var config = {
-					'production': '/api/'
-				};
-
-				return config;
-			}
-		};
-	});
+  'use strict';
+  angularAMD.factory('endpointConfiguration', function () {
+    return {
+      getEndpointUrl: function () {
```

Comment:
Yes, this is good idea - I renamed method to `getEndpoint` and `production` to `getUrl`.